src/fbw/src/interface/SimConnectInterface.cpp

```diff
@@ -892,6 +892,7 @@ void SimConnectInterface::simConnectProcessEvent(const SIMCONNECT_RECV_EVENT* ev
     }
 
     case Events::AUTOPILOT_ON: {
+      simInputAutopilot.AP_engage = 1;
       cout << "WASM: event triggered: AUTOPILOT_ON (currently inop)" << endl;
```

Comment:
question: Should the text be modified as this is no longer inop?

Reply:
Good spot!